modules/core-updates/8.0.md

```diff
@@ -57,233 +66,734 @@ To maintain the behavior of 1.7 and load the child theme's assets, `use_parent_a
 
 * In `checkout/_partials/steps/payment.tpl`, the ID `payment-form` has [been changed](https://github.com/PrestaShop/PrestaShop/pull/25069) to `payment-{$option.id}-form`.
 * Empty state messages are now shown inside the block (as opposed to the top of the page) in pages like order history, addresses, etc ([read more](https://github.com/PrestaShop/PrestaShop/pull/26806)).
+* The theme [now supports](https://github.com/PrestaShop/classic-theme/pull/21) a new password policy introduced [here](https://github.com/PrestaShop/PrestaShop/pull/28127/)
 
 ### Back office interface
 
 The Bootstrap version used in Symfony-based Back office pages has been updated from 4.0 to 4.4.
 
+### Github Project
+
+System-specific files, like IDE configuration and generated ones by OS, are [no longer ignored by the repository](https://github.com/PrestaShop/PrestaShop/pull/28623). They should to be added to the [global .gitignore](https://github.com/PrestaShop/PrestaShop/pull/28623#issuecomment-1144575464).
```

Comment:
Not sure if this is really worth mentioning? And personally I think it was a wrong move - people will forget and these files will be accidentally added into the project. It has already happened.

Reply:
It's the standard in other projects. Sometimes we can make developers' life easier, sometimes, we can educate them :-)